src/sonic-device-data/tests/platform_json_checker

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+try:
+    import re
+    import sys
+    import glob
+    import json
+except ImportError as e:
+    raise ImportError (str(e) + "- required module not found")
+try:
+    basestring
+except NameError:
+    basestring = str
```

Comment:
`basestring` has been removed in Python3. We will be migrating to Python 3 soon. Can you please add a TODO comment to remove this for Python 3 and just change to `str`?

Reply:
#done

Reply:
#done